scripts/regenerate.sh

```diff
@@ -38,39 +38,52 @@ echo "go install cmd/protoc-gen-go-grpc"
 (cd cmd/protoc-gen-go-grpc && go install .)
 
 echo "git clone https://github.com/grpc/grpc-proto"
-git clone --quiet https://github.com/grpc/grpc-proto ${WORKDIR}/grpc-proto
+git clone --quiet https://github.com/grpc/grpc-proto "${WORKDIR}/grpc-proto"
 
 echo "git clone https://github.com/protocolbuffers/protobuf"
-git clone --quiet https://github.com/protocolbuffers/protobuf ${WORKDIR}/protobuf
+git clone --quiet https://github.com/protocolbuffers/protobuf "${WORKDIR}/protobuf"
 
 # Pull in code.proto as a proto dependency
-mkdir -p ${WORKDIR}/googleapis/google/rpc
+mkdir -p "${WORKDIR}/googleapis/google/rpc"
 echo "curl https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto"
-curl --silent https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto > ${WORKDIR}/googleapis/google/rpc/code.proto
+curl --silent https://raw.githubusercontent.com/googleapis/googleapis/master/google/rpc/code.proto > "${WORKDIR}/googleapis/google/rpc/code.proto"
 
-mkdir -p ${WORKDIR}/out
+source ./scripts/protoc_installer.sh
+download_protoc $WORKDIR
+
+export PATH="$PATH:$GOBIN"
+
+PROTOC_PATH="/usr/local/bin"
+if [[ ":$PATH:" != *":$PROTOC_PATH:"* ]]; then
+  export PATH="$PATH:$PROTOC_PATH"
+  echo "protoc added to your PATH. You might need to open a new terminal"
+else
+  echo "protoc already appears to be on your PATH"
+fi
```

Comment:
We shouldnt be needing this. Ideally we want the `download_protoc` to be able to install binary to PATH